src/main/java/org/hibernate/search/develocity/plugins/SurefireConfiguredPlugin.java

```diff
@@ -26,14 +29,43 @@ protected void configureTest(GoalMetadataProvider.Context context) {
 			dependsOnConfigurableJavaExecutable( inputs, context, "jvm", isSkipped( context ),
 					JavaVersions::forJavaExecutable );
 		} );
+
+		// Develocity handles environment variables as a big bulk by default,
+		// which won't work if some variables point to absolute paths.
+		// Let's try to mimic system properties handling instead.
+		// NOTE: ignoring with inputs.ignore( "environmentVariables" ) doesn't work for some reason...
```

Comment:
`inputs.ignore( "environmentVariables" )` doesn't work, obviously:

```
Goal execution marked as not cacheable: Build caching was not enabled for this goal execution because properties were declared both as input and ignored: [environmentVariables].
```